docs/getting-started/README.md

```diff
@@ -51,15 +51,13 @@ $ npx buidler
   Quit
 ```
 
-Let’s create the sample project and go through the steps to try out the sample task and compile, test and deploy the sample contract.
+Let’s create the sample project and go through the steps to try out the sample task and compile, test and deploy the sample contract. 
 
-The sample project uses the `buidler-truffle5`, which makes Buidler compatible with
-tests built for Truffle. You can learn more about it [in this guide](../guides/truffle-testing.md). 
-For now, all you need to know is that you may need to install some dependencies with
- 
-```bash
-npm install --save-dev @nomiclabs/buidler-truffle5 @nomiclabs/buidler-web3 web3
-```
+The sample project will ask you to install `buidler-waffle` and `buidler-ethers`, which makes Buidler compatible with tests built for Waffle. You can learn more about it [in this guide](../guides/waffle-testing.md). 
+
+::: tip
+You can install those dependencies manually with `npm install --save-dev @nomiclabs/buidler-waffle ethereum-waffle chai @nomiclabs/buidler-ethers ethers`
```

Comment:
What I was trying to saying was:

In case you missed the installation of those depencies you can install them manually with...

Reply:
let's change it to

>Buidler will let you know how, but in case you missed it you can install them with `npm install --save-dev @nomiclabs/buidler-waffle ethereum-waffle chai @nomiclabs/buidler-ethers ethers`